<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>136</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="fileNameEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="openFileButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>37</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="importInfoButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>导入至教师信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
